src/net.h

```diff
@@ -87,6 +87,7 @@ static const bool DEFAULT_BLOCKSONLY = false;
 static const int64_t DEFAULT_PEER_CONNECT_TIMEOUT = 60;
 
 static const bool DEFAULT_FORCEDNSSEED = false;
+static const bool DEFAULT_DNSSEED = true;
```

Comment:
Mandatory nit involving constants (feel free to ignore): If you're going to add this, is it worth updating `init.cpp` where `argsman.AddArg` is called? Or is it worth not adding this constant in the first place (I'm not sure how much clarity it adds)?

Reply:
Good call on updating `argsman.AddArg` - I've pushed that up.

The benefit to adding this constant is that it increases the likelihood that all invocations use a consistent default.